docs/whats-new/zowe-v3-frequently-asked-questions.md

```diff
@@ -0,0 +1,123 @@
+# Zowe V3 Frequently Asked Questions
+
+## General questions
+
+1. When will Zowe V3 be released?
+  
+   Zowe V3 is scheduled to be released on Sept. 28, 2024. Any changes to the plan will be communicated via all available channels.
+
+2. When will Zowe V3 be available for preview?
+  
+   Currently, Zowe V3 pre-release versions are available for the PAX, Zowe CLI, Zowe Client Node.js SDKs, and Zowe Explorer for Visual Studio Code. **[Intellij? Zowe Desktop?]**
```

Comment:
@balhar-jakub - what about IntelliJ and Zowe Desktop?

Reply:
I believe that it's now available also for Intellij.